README.md

```diff
@@ -0,0 +1,33 @@
+# DamagingBlocks
+
+## Goal: 
+To allow modders to create blocks which cause damage to players upon physically touching them or destroy objects which land on the blocks (i.e. Lava)
```

Comment:
Maybe a second example could be cacti? Which actually could be great fun little thing to try implementing in a later task! :-)